### ISSUE TITLE
Title music runs out

### ISSUE THREAD

**Issue by Randalf98** [tizian/Cendric2#30]
Music in Main Menue runs out and doesn't restart.

**Comment by Ironbell** [comment]
That's intended by the composer.

**Comment by Arquata** [comment]
When the title music has run out and I start the game, there isn't any sound or music either, I guess this is not intended?

**Comment by Ironbell** [comment]
I'll check it.

**Comment by Ironbell** [comment]
Works for me. I let the music run out and started a new game. Could you specify the exact steps if you can reproduce it?

**Comment by Arquata** [comment]
I wasn't able to reproduce it so far, but I started the game and went to settings to put sound and music volume a bit down and switched the language to German, went back to the main menu and had the game just open for a while like this and did something else. When I started a new game there was no ingame sound or music (which I didn't realize at first because of other music on my computer)

But I just realized this also could have been a problem with my sound mixer which I already had with another program. So sorry if this is just another problem with my computer :D If it reappears and it is *not* the sound mixer I'll tell you.
